srcpkgs/k0s/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'k0s'
+pkgname=k0s
+version=0.9.1
+revision=1
+build_style=go
+go_import_path=github.com/k0sproject/k0s
+hostmakedepends="which golangci-lint"
+depends="kubernetes>1.20_1 containerd>=1.3.9_1"
+short_desc="Zero Friction Kubernetes"
+maintainer="Enno Boland <gottox@voidlinux.org>"
+license="Apache-2.0"
+homepage="https://k0sproject.io/"
+distfiles="https://github.com/k0sproject/$pkgname/archive/v$version.tar.gz"
+checksum=d67c48a61370610554d15fc61e23ac6b819f73c9dcb2c4d19fdde494e3d55dc9
+system_accounts="_k0s"
+_k0s_homedir="/var/lib/k0s"
+_k0s_shell="/bin/nologin"
+_k0s_descr="k0s server user"
+make_dirs="/var/lib/k0s 0750 _k0s _k0s"
+conf_files="/etc/k0s/k0s.conf"
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" k0s"
+fi
+
+pre_build() {
+	# Make sure files are generated correctly before actually building
+	make EMBEDDED_BINS_BUILDMODE=none generate-bindata
+}
+
+post_install() {
+	local K0S=$DESTDIR/usr/bin/k0s
+	if [ "$CROSS_BUILD" ]; then
+		K0S=k0s
+	fi
```

Comment:
You could try and use `vtargetrun` instead of depending on itself.

Reply:
I'm not sure what's the prettier solution here.

Reply:
Changed to `vtargetrun`.